zerver/models.py

```diff
@@ -2509,6 +2509,19 @@ def __str__(self) -> str:
                                                self.address or list(self.users.all()),
                                                self.scheduled_timestamp)
 
+class MissedMessageEmailAddress(models.Model):
+    EXPIRY_SECONDS = 60 * 60 * 24 * 5
+
+    message = models.ForeignKey(Message, on_delete=CASCADE)  # type: Message
+    user_profile = models.ForeignKey(UserProfile, on_delete=CASCADE)  # type: UserProfile
+    email_token = models.CharField(max_length=34, unique=True, db_index=True)  # type: str
+
+    timestamp = models.DateTimeField(db_index=True, default=timezone_now)  # type: datetime.datetime
```

Comment:
Let's add a comment above this field making clear that this is the timestamp the missed-message email was sent, and that it expires EXPIRY_SECONDS after that time, since one could imagine just reading this model that the convention had the expiry time included here.

Reply:
Added, in the first commit.